docs/source/documentation/fibertube_tracking.rst

```diff
@@ -0,0 +1,339 @@
+Introduction to the Fibertube Tracking environment through an interactive demo.
+====
+
+In this demo, you will be introduced to the main scripts of this project
+as you apply them on simple data. Our main objective is better
+understand and quantify the fundamental limitations of tractography
+algorithms, and how they might evolve as we approach microscopy
+resolution where individual axons can be seen. To do so, we will be
+evaluating tractography's ability to reconstruct individual white matter
+fiber strands at simulated extreme resolutions (mimicking "infinite"
+resolution).
+
+Terminology
+-----------
+
+Here is a list of terms and definitions used in this project.
+
+General:
+
+-  Axon: Bio-physical object. Portion of the nerve cell that carries out
+   the electrical impulse to other neurons. (On the order of 0.1 to 1um)
+-  Streamline: Virtual object. Series of 3D coordinates approximating an
+   underlying fiber structure.
+
+Fibertube Tracking:
+
+-  Fibertube: Virtual representation of an axon. Tube obtained from
+   combining a diameter to a streamline.
+-  Centerline: Virtual object. Streamline passing through the center of
+   a tubular structure.
+-  Fibertube segment: Cylindrical segment of a fibertube that comes as a
+   result of the discretization of its centerline.
+-  Fibertube Tractography: The computational tractography method that
+   reconstructs fibertubes. Contrary to traditional white matter fiber
+   tractography, fibertube tractography does not rely on a discretized
+   grid of fODFs or peaks. It directly tracks and reconstructs
+   fibertubes, i.e. streamlines that have an associated diameter.
+
+.. image:: https://github.com/user-attachments/assets/0286ec53-5bca-4133-93dd-22f360dfcb45
+   :alt: Fibertube visualized in 3D
+
+Methodology
+-----------
+
+This project can be split into 3 major steps:
+
+-  Preparing ground-truth data: We will be using the ground-truth of
+   simulated phantoms of streamlines, along with a diameter (giving us
+   fibertubes) and ensuring that they are void of any collision, i.e.
+   fibertubes in the simulated phantom should not intersect one another.
+   This is physically impossible to respect the geometry of axons.
+-  Tracking and experimentation: We will perform fibertube tracking on
+   our newly formed set of fibertubes with a variety of parameter
+   combinations.
+-  Evaluation metrics computation: By passing the resulting tractogram
+   through different evaluation scripts (like Tractometer), we will
+   acquire connectivity and fiber reconstruction scores for each of the
+   parameter combinations.
+
+Preparing the data
+------------------
+
+To download the data required for this demo, open a terminal and activate your
+scilpy virtual environment. Then, navigate to the scilpy repository on your
+computer and enter the command: ``pytest -v`` **\***.
```

Comment:
To get the file needed you need to run this command line:

wget https://scil.usherbrooke.ca/scil_test_data/dvc-store/files/md5/82/248b4888a63b0aeffc8070cc206995 -O others.zip

Reply:
then unzip ...